include/circt/Dialect/Calyx/Calyx.td

```diff
@@ -36,6 +36,7 @@ def CalyxDialect : Dialect {
     /// Register all Calyx attributes.
     void registerAttributes();
   }];
+  let dependentDialects = ["circt::hw::HWDialect"];
```

Comment:
While not entirely necessary, I'd add a comment on *why* we are dependent on this.